README.md

```diff
@@ -9,22 +9,42 @@ The plan is
 
 # Using `taichi_elements` in Python
  - Install [taichi](https://github.com/taichi-dev/taichi) with `pip`: `python3 -m pip install taichi`
+ - Execute `python3 download_ply.py` to download model files used by the demos
  - `python3 demo/demo_2d.py` and you will see
  <img src="https://github.com/yuanming-hu/public_files/raw/master/graphics/elements/demo_2d.gif">
- 
+
  - `python3 demo_3d.py` and you will see a 3D simulation visualized in 2D.
 
 ## To simulate and render an example 3D scene with Python
  - Make sure you have a modern NVIDIA GPU (e.g. GTX 1080 Ti)
  - Download [`taichi.ply`](https://github.com/taichi-dev/taichi_elements_blender_examples/releases/download/ply/taichi.ply) and run `python3 demo_3d_letters.py` (wait for at least 10 frames)
    - A binary particle folder with a timestamp in its time (e.g., `sim_2020-07-27_20-55-48`) will be created under the current folder.
- - `python3 engine/render_particles [particle_input_folder] [begin] [end] [step] [render_output_folder]`
-   - E.g., `python3 engine/render_particles.py sim_2020-07-27_20-55-48/ 0 100 1 frames`
+ - Example:
+
+   - ```python
+     python engine/render_particles
+     -i ./path/to/particles #input directory
+     -b 0 -e 400 -s 1 # render 0~400 frame with step 1
+     -o ./path/output
+     --gpu-memory 20 # 20GB GPU memory
+     -M 460 # at maximum 460 million particles
+     --shutter-time 0.0 # How much Motion blur effect.
+     # Notice Memory usage increase when shutter_time increases
+     -r 128 # rendering spatial grid dimension
+     ```
```

Comment:
I don't think you can directly execute this. At least you need `\` to concatenate the lines.

Reply:
Hi, I think reader would not bother adding the `\`(or squash them into one line ) themselves because the annotation here. They need to remove the annotation to run the command. So I don't think we should add the `\` actually(for the sake of beauty). Some IDLE(like Pycharm) will even automatically add the `\` for you😂

Reply:
I think you should try to create a piece of example code that people can directly copy-paste into the command line and then execute, after the paths are replaced. I don't think in bash these multi-line commends can be automatically concatnated.